src/lxc/console.c

```diff
@@ -121,27 +121,29 @@ struct lxc_tty_state *lxc_console_sigwinch_init(int srcfd, int dstfd)
 	ts->masterfd = dstfd;
 	ts->sigfd = -1;
 
-	/* add tty to list to be scanned at SIGWINCH time */
-	lxc_list_add_elem(&ts->node, ts);
-	lxc_list_add_tail(&lxc_ttys, &ts->node);
+	if (isatty(srcfd)) {
```

Comment:
From a coding style point of view it would be nicer to just do:

```
if (!isatty(srcfd)) {
        INFO("fd %d does not refer to a tty device", srcfd);
        return ts;
}
```
This lets you avoid the additional indentation level afterwards.